url-updater/src/main/java/com/devonfw/tools/ide/url/updater/jmc/JmcUrlUpdater.java

```diff
@@ -0,0 +1,39 @@
+package com.devonfw.tools.ide.url.updater.jmc;
+
+import com.devonfw.tools.ide.url.model.folder.UrlVersion;
+import com.devonfw.tools.ide.url.updater.GithubUrlUpdater;
+
+/**
+ * {@link GithubUrlUpdater} for java mission control.
+ */
+public class JmcUrlUpdater extends GithubUrlUpdater {
+
+  @Override
+  protected String getTool() {
+
+    return "jmc";
+  }
+
+  @Override
+  protected String getGithubOrganization() {
+
+    return "adoptium";
+  }
+
+  @Override
+  protected String getGithubRepository() {
+
+    return "jmc-build";
```

Comment:
latest tag is  9.0.0-SNAPSHOT but there is no such release. Can you filter out SNAPSHOT versions?

Reply:
While testing locally the SNAPSHOT version already got filtered, or do you mean something else? 
<img width="631" alt="image" src="https://github.com/devonfw/ide/assets/44358321/1fefd33b-8bf3-472b-8413-0f812da9b2a5">

Reply:
Thanks for checking. Resolved.